README.md

```diff
@@ -96,6 +96,40 @@ services:
       - webchangess
 ```
 
+## Testing
+
+You can use
+``` shell
+docker compose exec webchanges sh
+cd /data/webchanges
+```
+and then
+``` shell
+webchanges --urls jobs.yaml --config config.yaml --cache cache.db --list
```

Comment:
`--cache` is deprecated and should be replaced by `--database` and `cache.db` with `snapshots.db` (https://github.com/mborsetti/webchanges/releases/tag/v3.22)

```suggestion
webchanges --urls jobs.yaml --config config.yaml --database snapshots.db --list
```

Reply:
Ah, interesting. I have taken the command from the old `crontabfile` which was not updated automatically.

Question (only semi-related to this PR): Do we need a migration step or at least something in the README? Looking at your link (the webchanges v3.22 release), they say that `--cache` is deprecated and will be removed in a future release so endless `docker compose pull` will eventually lead to malfunctioning.

Also...

![image](https://github.com/user-attachments/assets/18769425-3488-4a0a-bbc0-bb28bb209935)

:wink: